climada/util/calibrate/base.py

```diff
@@ -158,6 +159,81 @@ def plot_impf_set(self, **plot_kwargs):
         """
         return self.impf_set.plot(**plot_kwargs)
 
+    def plot_impf_variability(
+        self,
+        cost_func_diff: float = 0.1,
+        p_space_df: Optional[pd.DataFrame] = None,
+        plot_haz: bool = True,
+        **plot_kwargs
+    ):
+
+        """Plot impact function variability with parameter combinations of
+        almost equal cost function values
+
+        Args:
+            cost_func_diff (float, optional): Max deviation from optimal cost
+            function value (as fraction). Defaults to 0.1 (i.e. 10%).
+            p_space_df (pd.DataFrame, optional): parameter space. Defaults to None.
+            plot_haz (bool, optional): Whether or not to plot hazard intensity
+            distibution. Defaults to False.
+        """
+
+        haz_type = self.input.hazard.haz_type
+        if p_space_df is None:
+            p_space_df = self.output.p_space_to_dataframe()
+
+        # Retrieve list of parameters required for creating impact functions
+        # and remove the dimension 'Cost Function'.
+        params = p_space_df.columns.tolist()
+        try:
+            params.remove('Cost Function')
+        except ValueError:
+            pass
+
+        # Retrieve parameters of impact functions with cost function values
+        # within 'cost_func_diff' % of the best estimate
+        max_cost_func_val = p_space_df['Cost Function'].min()*(1+cost_func_diff)
+        params_within_range = p_space_df.loc[p_space_df['Cost Function'] <=
+                                             max_cost_func_val,params]
+
+        # Initialize figure
+        fig,ax = plt.subplots()
+
+        #Plot best-fit impact function
+        best_impf = self.impf_set.get_func(haz_type=haz_type)[0]
+        ax.plot(best_impf.intensity,best_impf.mdd*best_impf.paa*100,
+                color='tab:blue',lw=2,zorder=3,label='best fit',**plot_kwargs)
```

Comment:
Using `**plot_kwargs` like this does not let you update the default parameters like `color` or `lw`. I suggest pulling the default arguments from `plot_kwargs` using `pop`, as I did in the other plots of the module: https://github.com/CLIMADA-project/climada_python/pull/692/files#diff-c11ede847f0d2a20c8d179c645028e1b00b8a81735677b29c3538edb7446ed02R208

Reply:
Nice!